tools/malwarejail/env/wscript.js

```diff
@@ -2,15 +2,15 @@
     wscript.js - simulates WScript (Windows scripting host) environment
 */
 
-const { Blob } = require("node:buffer");
-const URL = require('url').URL;
+const { resolveObjectURL } = require('node:buffer');
+const { win32 } = require('path');
 
 util_log("Preparing sandbox to emulate WScript environment.");
 _wscript_saved_files = {};
 _wscript_urls = [];
 _wscript_wmis = [];
 _wscript_objects = [];
-_pw32 = require('path').win32;
+_pw32 = win32;
```

Comment:
We could just use "win32" everywhere we use "_pw32", maybe next cleanup session.